frame/collective/src/lib.rs

```diff
@@ -669,11 +693,30 @@ pub mod pallet {
 	#[pallet::genesis_build]
 	impl<T: Config<I>, I: 'static> GenesisBuild<T, I> for GenesisConfig<T, I> {
 		fn build(&self) {
+			<UpgradedToTripleRefCount<T, I>>::put(false);
```

Comment:
I would expect to put `true`, if the genesis is build it means there wasn't any storage before, thus we don't need any migration.

Reply:
Make sense, Done :+1: